src/Files.App/Data/Contexts/Multitasking/MultitaskingContext.cs

```diff
@@ -39,6 +39,7 @@ public MultitaskingContext()
 		private void AppInstances_CollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
 		{
 			UpdateTabCount();
+			AppLifecycleHelper.SaveSessionTabs();
```

Comment:
Can we delete any other code that calls `SaveSessionTabs()`?

Reply:
Probably

Reply:
They might be worth keeping because `AppInstances_CollectionChanged` is only called when a tab is added/removed. This is fine for a single instance, but it can cause issues with multiple windows. That said, if we add support for saving the tab list for multiple windows, I think we can rely on this one call.